matplotlibcpp.h

```diff
@@ -154,6 +155,63 @@ namespace matplotlibcpp {
 		return res;
 	}
 
+	template< typename Numeric>
+	bool hist(const std::vector<Numeric>& y, long bins=10,std::string color="b", double alpha=1.0){
+
+		PyObject* ylist = PyList_New(y.size());
+		
+		PyObject* kwargs = PyDict_New();
+		PyDict_SetItemString(kwargs, "bins" ,PyFloat_FromDouble(bins));
+		PyDict_SetItemString(kwargs,"color",PyString_FromString(color.c_str()));  
+	  PyDict_SetItemString(kwargs, "alpha" ,PyFloat_FromDouble(alpha));
+		
+		for(size_t i = 0; i < y.size(); ++i) {
+			PyList_SetItem(ylist, i, PyFloat_FromDouble(y.at(i)));
+		}
+
+		PyObject* plot_args = PyTuple_New(1);
+
+		PyTuple_SetItem(plot_args, 0, ylist);
+
+
+		PyObject* res = PyObject_Call(detail::_interpreter::get().s_python_function_hist, plot_args, kwargs);
+
+
+		Py_DECREF(ylist);
+		Py_DECREF(plot_args);
```

Comment:
It's looks like it's working correctly like this, no negative values, and seems to be coherent with the doc: https://docs.python.org/2.7/c-api/refcounting.html#c.Py_DECREF